cmd/gocritic/check.go

```diff
@@ -162,11 +162,10 @@ func (p *program) checkFile(f *ast.File) {
 				if err, ok := r.(error); ok {
 					log.Printf("%s: error: %v\n", c.Info.Name, err)
 					panic(err)
-				} else {
```

Comment:
Which check reports this line?

Reply:
It's from `revive` ([`superfluous-else`](https://revive.run/r#superfluous-else)):
```
cmd/gocritic/check.go:165:12: superfluous-else: if block ends with call to panic function, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (revive)
                                } else {
                                        // Some other kind of run-time panic.
                                        // Undo the recover and resume panic.
                                        panic(r)
                                }
```